conf/app.routes

```diff
@@ -21,7 +21,9 @@ GET         /undeliverable-email                    controllers.EmailController.
 GET         /request-cash-transactions/v2          controllers.SelectTransactionsController.onPageLoad()
 POST        /request-cash-transactions/v2          controllers.SelectTransactionsController.onSubmit()
 GET         /selected-cash-transactions            controllers.SelectedTransactionsController.onPageLoad()
+POST        /selected-cash-transactions            controllers.SelectedTransactionsController.onSubmit()
```

Comment:
Better to give a line break after this line and before line 23 to keep both GET and POST together and easier to find